extensions/wikia/PortableInfobox/querypage/AllinfoboxesQueryPage.php

```diff
@@ -104,9 +104,14 @@ public function reallyDoQuery( $limit = false, $offset = false ) {
 			} );
 
 		return array_filter( $result, function ( $tmpl ) {
-			$data = PortableInfoboxDataService::newFromPageID( $tmpl[ 'pageid' ] )->getData();
+			$title = Title::newFromID( $tmpl[ 'pageid' ] );
+			if ( $title && $title->exists() && !$title->isSubpage() ) {
+				$data = PortableInfoboxDataService::newFromTitle( $title )->getData();
```

Comment:
From my perspective it depends on the scope of such usage among the existing infoboxes. 
I don't think that it's a blocker; However, I suggest to validate this either as a follow-up task or part of testing (preferred) should we go with the approached proposed with this PR.